### ISSUE TITLE
ERROR: '/home/jason/.local/share/tmux/plugins/tmux-fzf-url/fzf-url.sh '' screen' returned 2

### ISSUE THREAD

**Issue by eggbean** [wfxr/tmux-fzf-url#32]
It's been a while since I used this, but now everything works as expected until I select a URL and then I get this error:

    '/home/jason/.local/share/tmux/plugins/tmux-fzf-url/fzf-url.sh '' screen' returned 2

What does this mean?

**Comment by Tony-Sol** [comment]
Almost same here, got
```
~/.tmux/pluginstmux-fzf-url/fzf-url.sh '' screen ''' returned 1
```

**Comment by eggbean** [comment]
I don't think I have tried using this since I posted this, but I just now thought, why does it mention "screen"? Does this plugin work with screen as well or something? Is it trying to pass the command to screen rather than tmux?

**Comment by Tony-Sol** [comment]
No, it doesn't use screen (if you mean screen as terminal multiplexer)
In my case, there is error in `read` processing, 'cause i've got `read failed 9: Bad file descriptor` after some research

**Comment by eggbean** [comment]
The plugin is actually working perfectly fine for me.

These are my settings in my `tmux.conf`:

```
# tmux-fzf-URL
set -g @plugin 'wfxr/tmux-fzf-url'
set -g @fzf-url-fzf-options '-w 50% -h 60% --multi -0 --no-preview --layout reverse --prompt "url: " --no-info --pointer ÔùÅ --color "prompt:#bd93f9,pointer:#bd93f9"'
set -g @fzf-url-history-limit '50'
```

**Comment by eggbean** [comment]
I just remembered something which may help: When I was having problems like this with a couple of tmux plugins (I can't remember if one of them was this one), it worked on some computers but now on other, and I realised that the ones it didn't work on were Debian-based. Make sure that you have `gawk` installed, so that `awk` is aliased to `gawk` and not `mawk`.

**Comment by bob-rescale** [comment]
Getting the same error after not using this for a while, and the plugin doesn't seem to work (I did have `mawk` and switched to `gawk`, but that doesn't seem to have been the issue)
